client/state/data-layer/wpcom/signup/verticals/index.js

```diff
@@ -25,14 +25,16 @@ export const requestVerticals = action =>
 			path: '/verticals',
 			query: {
 				search: action.search.trim(),
+				site_type: action.siteTypeId,
```

Comment:
Sorry, I should have been more explicit [in my comment](https://github.com/Automattic/wp-calypso/pull/33209#discussion_r286295904), I was referring to the value we send to the backend only.

It'd be okay, as @spen mentioned, to keep the slugs as keys for clarity.

Reply:
Ah that's on me - should be okay now